packages/council-cli/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@council/cli",
   "version": "0.0.1",
   "license": "MIT",
-  "source": "src/index.ts",
+  "source": "src/council.ts",
```

Comment:
ooh i like this, where does the help text come up?

Reply:
Thanks! It comes up in all the command names here. They used to say `index config [action]`